### ISSUE TITLE
Fix image build failure due to incorrect group in Dockerfile for Coherence 12.2.1.4.0

### ISSUE THREAD

**Issue by dhirupandey** [oracle/docker-images#1486]
When running the mvn clean install command, the build fails as shown below:
`

    mvn clean install
    [INFO] Scanning for projects...
    [INFO]
    [INFO] ------------------------------------------------------------------------
    [INFO] Building Coherence Docker Distribution 12.2.1-4-0
    [INFO] ------------------------------------------------------------------------
    [INFO]
    [INFO] --- maven-clean-plugin:2.5:clean (default-clean) @ coherence-docker ---
    [INFO] Deleting /Users/dhiru/work/coherence/docker/docker-images/OracleCoherence/dockerfiles/12.2.1.4.0/target
    [INFO]
    [INFO] --- maven-assembly-plugin:2.5.3:single (distribution) @ coherence-docker ---
    [INFO] Reading assembly descriptor: src/main/assemble/coherence.xml
    [INFO] Copying files to /Users/dhiru/work/coherence/docker/docker-images/OracleCoherence/dockerfiles/12.2.1.4.0/target/docker-build
    [WARNING] Assembly file: /Users/dhiru/work/coherence/docker/docker-images/OracleCoherence/dockerfiles/12.2.1.4.0/target/docker-build is not a regular file (it may be a directory). It cannot be attached to the project build for installation or deployment.
    [INFO]
    [INFO] --- docker-maven-plugin:0.20.1:build (package) @ coherence-docker ---
    [INFO] Building tar: /Users/dhiru/work/coherence/docker/docker-images/OracleCoherence/dockerfiles/12.2.1.4.0/target/docker/oracle/coherence/12.2.1.4.0/tmp/docker-build.tar.bz
    [INFO] DOCKER> [oracle/coherence:12.2.1.4.0]: Created docker-build.tar.bz in 19 seconds
    [ERROR] DOCKER> Unable to build image [oracle/coherence:12.2.1.4.0] : The command '/bin/sh -c chmod a+xr /u01 && chown oracle:oracle -R /u01' returned a non-zero code: 1 [The command '/bin/sh -c chmod a+xr /u01 && chown oracle:oracle -R /u01' returned a non-zero code: 1 ]
    [INFO] ------------------------------------------------------------------------
    [INFO] BUILD FAILURE
    [INFO] ------------------------------------------------------------------------
    [INFO] Total time: 53.782 s
    [INFO] Finished at: 2020-01-03T15:05:29-08:00
    [INFO] Final Memory: 19M/257M
    [INFO] ------------------------------------------------------------------------
    [ERROR] Failed to execute goal io.fabric8:docker-maven-plugin:0.20.1:build (package) on project coherence-docker: Unable to build image [oracle/coherence:12.2.1.4.0] : The command '/bin/sh -c chmod a+xr /u01 && chown oracle:oracle -R /u01' returned a non-zero code: 1 -> [Help 1]
    [ERROR]
    [ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
    [ERROR] Re-run Maven using the -X switch to enable full debug logging.
    [ERROR]
    [ERROR] For more information about the errors and possible solutions, please read the following articles:
    [ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException

`

The error comes from executing the following command in Dockerfile:
chown oracle:oracle -R /u01

The group should be root instead of oracle

```
docker>git diff Dockerfile 
diff --git a/OracleCoherence/dockerfiles/12.2.1.4.0/src/main/docker/Dockerfile b/OracleCoherence/dockerfiles/12.2.1.4.0/src/main/docker/Dockerfile
index a44d7d3..c1416c7 100644
--- a/OracleCoherence/dockerfiles/12.2.1.4.0/src/main/docker/Dockerfile
+++ b/OracleCoherence/dockerfiles/12.2.1.4.0/src/main/docker/Dockerfile
@@ -50,7 +50,7 @@ RUN mkdir -p /u01 && \
 COPY $FMW_PKG install.file oraInst.loc /u01/
 RUN chmod a+xr /u01 && \
-    chown oracle:oracle -R /u01
+    chown oracle:root -R /u01
 USER oracle
```

**Comment by Djelibeybi** [comment]
You don't explain what the build failure is here. Just what the change you expect to see is. Can you provide actual detail of what is failing?

**Comment by rjeberhard** [comment]
@dhirupandey, just a heads-up that `chown -R` will nearly double the final size of your Docker image as it touches every file.  I haven't looked to see if you've already implemented the multistage build pattern in your Dockerfile, but I would recommend it and then change the ownership on the `COPY`.

**Comment by dhirupandey** [comment]
@rjeberhard - Actually this change is in the part of multistage build that is not used for creating the image. That said we did find another place in the Dockerfile where we are doing that. I plan to fix that ... @thegridman and I were just talking about this a few minutes back.

Thank you for your insight and feedback. FWIW I did find an interesting tool called dive https://github.com/wagoodman/dive that helps you narrow down which command adds how much to the size of the image
